unreleased/detect-pod.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

Comment:
This isn't a breaking change, is it?  It doesn't change exported APIs at all.  It seems more like an enhancement to me.

Reply:
Agreed, thanks for the suggestion